playbooks/support/responding.md

```diff
@@ -79,9 +79,9 @@ One of the on-call members handles internal and external communication for the i
 - Meaningful updates, discoveries, or milestones (e.g. changes in availability or resolution) are added to the Jira
   Ops ticket by the assignee.
 - Any incident with public impact should be communicated externally via the status page. Communication does not
-  have to be detailed. The most important thing is acknowledging issues and communicating progress towards a
-  resolution. Use the status page integration on the Jira Ops ticket for posting updates and resolutions.
+  have to be detailed(Not every stage of the incident necessarily need to be communicated on status page). Use your judgement based on severity and duration of the incident. The most important thing is acknowledging issue, specifying the services affected and communicating key updatespdates and resolutions. Use the status page integration on the Jira Ops ticket for updating Status page.
```

Comment:
Thanks for clarifying this!

There are a few typos above (minor).

Maybe worth adding that these status page updates are oriented toward a _public_ audience (i.e., users and partners). Some may even be subscribed (like I do for certain providers, like Elasticsearch cloud), so receive an email about every incident and update.